src/Examples/Tensorboard.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+using SkiaSharp;
+using static TorchSharp.torch;
+
+namespace TorchSharp.Examples
+{
+    public static class Tensorboard
+    {
+        private static string imagePath = "TensorboardExample/Images";
+        internal static async Task Main(string[] args)
+        {
+            await DownloadExampleData("https://agirls.aottercdn.com/media/0d532b3f-0196-466a-96e1-7c6db56d0142.gif");
+            var device = cuda.is_available() ? CUDA : CPU;
+            var writer = utils.tensorboard.SummaryWriter("tensorboard");
+            Console.WriteLine($"Running Tensorboard on {device.type}");
+            AddText(writer);
+            AddImageUsePath(writer);
+            AddImageUseTensor(writer, device);
+            AddVideo(writer, device);
+        }
+
+        private static void AddText(Modules.SummaryWriter writer)
+        {
+            writer.add_text("AddText", "step_1", 1);
+            writer.add_text("AddText", "step_2", 2);
+            writer.add_text("AddText", "step_3", 3);
+            writer.add_text("AddText", "step_4", 4);
+            writer.add_text("AddText", "step_5", 5);
+        }
+
+        private static void AddImageUsePath(Modules.SummaryWriter writer)
+        {
+            var imagesPath = Directory.GetFiles(imagePath);
+            for (var i = 0; i < imagesPath.Length; i++) {
+                writer.add_img("AddImageUsePath", imagesPath[i], i);
+            }
+        }
```

Comment:
> Yet another comment -- TorchVision has APIs for reading and writing images from/to disk. It would make sense, I think, to use those in the example, since examples are helpful for showing users how to accomplish things. They already handle the RGB vs. gray-scale issues, etc.
> 
> Is there a reason those aren't useful here?

do you mean this?